common/lib/xmodule/xmodule/services.py

```diff
@@ -0,0 +1,19 @@
+"""
+Module contains various XModule/XBlock services
+"""
+from django.conf import settings
+
+class SettingsService(object):
+    """
+    Allow XBlocks to access the server settings
+    """
+    xblock_settings_bucket_selector = 'block_settings_key'
+
+    def get_settings_bucket(self, block, default=None):
```

Comment:
Is "None" the appropriate default here? (just because an unexpected None can cause more issues than an unexpected False)